src/views/start-page/start-page.less

```
@@ -71,50 +76,139 @@
     flex: 1 1 40%;
 
     .top-content {
+      display: flex;
       flex: 1 1 auto;
-      background-image: url("../assets/landing-page-image.jpg");
+      background-image: url("../assets/dual.jpg");
       background-repeat: no-repeat;
       background-size: cover;
       background-position: center;
+
+      .top-content-overlay {
+        flex: 1 1 auto;
+        background: linear-gradient(135deg, rgba(0,152,68,0.3) 0%, transparent 100%);
+      }
     }
 
     .bottom-content {
       display: flex;
       justify-content: center;
       align-items: center;
-      flex: 0 0 100px;
+      flex: 0 0 130px;
       background: @colorPurple;
+      box-sizing: border-box;
 
-      button {
-        height: 55px;
-        width: 180px;
-        position: relative;
-        font-size: 20px;
-        padding: 10px 20px;
-        border: none;
-        background: @darkThemeFontColor;
-        color: @darkThemeBackgroundColor;
-        font-family: 'Source Sans Pro', sans-serif;
-        cursor: pointer;
-
-        &:active {
-          background: rgba(255,255,255,.7);
-        }
+      #feedback {
+        .form {
+          flex: 1 1 auto;
+          width: 100%;
+          height: 100%;
+          display: flex;
+          flex-direction: column;
+          justify-content: space-between;
+          align-items: center;
+          box-sizing: border-box;
 
-        &:focus {
-          outline: none;
+        .happy-container {
+          display: flex;
+          flex-direction: column;
+          align-items: center;
+          width: 80%;
+
+          p {
+            font-size: 1.2rem;
+          }
+
+          .happy-buttons {
+            display:flex;
+            justify-content: center;
+
+            input {
+              position: absolute;
+              opacity: 0;
+              cursor: pointer;
+              height: 0;
+              width: 0;
+            }
+
+            .happy, .sad {
```

separate selectors with newline for readability
```suggestion
            .happy,
            sad {